agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/JvmCompiler.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.init;
+
+import com.microsoft.applicationinsights.agent.bootstrap.diagnostics.PidFinder;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+class JvmCompiler {
+
+  // To disable C2 compiler during Application Insights set-up, for Java >= 9, do the following
```

Comment:
can you add a note how user can use `APPLICATIONINSIGHTS_EXPERIMENT_CLEAR_COMPILER_DIRECTIVES_AFTER_INITIALIZATION`?

Reply:
@heyamsthis It's an experimental feature that we would like to test with some users. Following the feedbacks, we could properly document this feature.

Reply:
@heyams It is necessary to add the JVM option explained from line 11 and set the `applicationinsights.experiment.clear-compiler-directives-after-initialization` property to `true`.